clients/cli/src/clap_app.rs

```diff
@@ -890,6 +894,14 @@ pub fn app<'a>(
                         .takes_value(false)
                         .help("Enables group member configurations in the mint. The mint authority must initialize the member."),
                 )
+                .arg(
+                    Arg::with_name("pausable")
+                        .long("pausable")
```

Comment:
nit: Can this be renamed for `enable-pause` for consistency?
```suggestion
                    Arg::with_name("enable_pause")
                        .long("enable-pause")
```

Reply:
Oh yes I missed this. Thanks!